storageconnector/pgconnector.go

```diff
@@ -5,48 +5,119 @@ import (
 	"fmt"
 	_ "github.com/lib/pq"
 	"regexp"
+	"strings"
 )
 
 type postgreStorage struct {
 	db *sql.DB
 }
 
-func ConnectToPg (username string, dbname string) {
+func ConnectToPg(username string, dbname string) {
 	fmt.Println("Opening database connection.")
-	command := fmt.Sprintf("user=%s dbname=%s sslmode=disable",username,dbname)
+	command := fmt.Sprintf("user=%s dbname=%s sslmode=disable", username, dbname)
 	db, err := sql.Open("postgres", command)
 
 	checkErr(err)
 	fmt.Println("Successfully opened database connection.")
-	db.Exec("CREATE TABLE IF NOT EXISTS features (index int, name text, description text, author text, lastupdate timestamptz, dependencies text);")
+	db.Exec("CREATE TABLE IF NOT EXISTS features (index int, name text, description text, author text, lastupdate timestamptz, dependencies text, snippet text);")
 }
 
-func (store* postgreStorage) disconnectFromPg() {
+func (store *postgreStorage) disconnectFromPg() {
 	store.db.Close()
 }
 
-func (store* postgreStorage) SearchMeta(name *regexp.Regexp) ([]FeatureMeta, error) {
-// TODO: implement (github issue #91)
-	return nil, nil
+func (store *postgreStorage) SearchMeta(name *regexp.Regexp) ([]FeatureMeta, error) {
+	var sql_query string
+	var dep_text string
+	var snippet string
+	sql_query = fmt.Sprintf("select * from features where name ~ %s", name)
+	rows, err := store.db.Query(sql_query)
+	checkErr(err)
+	defer rows.Close()
+	var fms []FeatureMeta
+	for rows.Next() {
+		var f FeatureMeta
+		var index int
+		err := rows.Scan(index, f.Name, f.Description, f.Author, f.UpdatedAt, dep_text, snippet)
```

Comment:
I don't get what you mean. Yes, Snippet is part of Feature. It's ignored here. Using _ doesn't work for the variadic Scan-function.